code/game/objects/items/toys.dm

```diff
@@ -1035,6 +1035,14 @@
 			cardUser.visible_message(span_notice("[cardUser] checks [cardUser.p_their()] card."), span_notice("The card reads: [cardname]."))
 		else
 			. += span_warning("You need to have the card in your hand to check it!")
+		. += span_notice("Right-click to flip it.")
+
+/obj/item/toy/cards/singlecard/attack_hand_secondary(mob/user, params)
+	. = ..()
+	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
+		return
+	Flip()
```

Comment:
You should make Flip a proc rather then a verb, since we're adding a less crummy way to achieve it

Reply:
Applied with https://github.com/tgstation/tgstation/pull/64200/commits/63532533bdb6e4644c16b7411440e83984b89d86